README.md

```diff
@@ -1,19 +1,28 @@
-# Class-Profile1A-Website
-Website for the SYDE 2025 1A Class Profile.
+# Class-Profile2A-Website
+Website for the SYDE 2025 2A Class Profile.
 
 ## Table of Contents
-- [SYDE2025-1A-Class-Profile-](#syde2025-1a-class-profile-)
+- [SYDE2025-2A-Class-Profile](#syde2025-1a-class-profile-)
   * [About The Project](#about-the-project)
-    + [Purpose](#purpose)
-    + [Languages, Tools and Libraries](#languages-tools-and-libraries)
+  * [Languages, Tools and Libraries](#languages-tools-and-libraries)
+  * [Build instruction](#build-instruction)
   * [License](#license)
   * [Contact](#contact)
 
 ## About The Project
-The SYDE class profile, was created to give an insight to the SYDE 2025 cohort's 1A term, similar to reports created
-by previous cohorts. We wanted to go a step above and create a website that would document not just our 1A term, but future terms as well.
+The SYDE class profile, was created to give an insight to the SYDE 2025 cohort's 2A term, similar to reports created
+by previous cohorts. We wanted to go a step above and create a website that would document not just our 2A term, but future terms as well.
 
-### Languages, Tools and Libraries
+## Build instruction 
+- On develop branch, clone the repo (git clone [URL])
+- cd into folder
+- Run the following commands
+  - ```yarn install```
```

Comment:
Merely a personal preference but I like npm install, u seem to be using npm commands later anyway

Reply:
npm install didn't really work for me. It was showing dependency errors. I used yarn and it worked fine.